test/jdk/java/foreign/TestClassLoaderFindNative.java

```diff
@@ -40,7 +40,7 @@
 import java.lang.foreign.ValueLayout;
 import org.testng.annotations.Test;
 
-import static java.lang.foreign.ValueLayout.JAVA_BYTE;
```

Comment:
Please leave this line alone - there's no need to create a difference here relative to upstream.

Reply:
Fixed.